### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dereuromark**
Use new 3.0 version of sniffer lib.

### CODE REVIEW

File: Spryker/Sniffs/Commenting/DocBlockTagGroupingSniff.php [hunk 1]
```diff
@@ -172,7 +172,7 @@ protected function checkBeginningOfDocBlock(PHP_CodeSniffer_File $phpCsFile, $do
             return;
         }
 
-        $fix = $phpCsFile->addFixableError('Expected no extra blank line before tags, got ' . ($diff - 1), $nextIndex);
+        $fix = $phpCsFile->addFixableError('Expected no extra blank line before tags, got ' . ($diff - 1), $nextIndex, 'NoExtraNewlineBeforeTags');
         if ($fix) {
```
Comment (darius-telycenas) [inline Spryker/Sniffs/Commenting/DocBlockTagGroupingSniff.php]: Shouldn't be inverse condition — `(!$fix)` — here?

Comment (dereuromark) [inline Spryker/Sniffs/Commenting/DocBlockTagGroupingSniff.php]: you are right. thx for the catch!

Comment (dereuromark) [inline Spryker/Sniffs/Commenting/DocBlockTagGroupingSniff.php]: Fixed in master @ https://github.com/spryker/code-sniffer/commit/70400d62b1b734b8fe908977657f185f2d1c3552

**Comment by dereuromark** [comment]
@darius-telycenas Only tests are erroring so far, because of some strange

> Uncaught TypeError: Argument 1 passed to PHPUnit\Framework\TestSuite::addTest() must implement interface PHPUnit\Framework\Test, instance of PHP_CodeSniffer\Tests\TestSuite given, called in phar:///home/travis/.phpenv/versions/7.0.21/bin/phpunit/phpunit/Framework/TestSuite.php

I went down from phpunit 6 to 5 to 4 with no difference...
Any ideas?

**Comment by darius-telycenas** [comment]
IMHO, `https://github.com/spryker/code-sniffer/tree/develop/Spryker/tests/` should be compared to `https://github.com/squizlabs/PHP_CodeSniffer/tree/master/tests/`.

```
<?php
class SprykerStandardTest extends PHPUnit_Framework_TestCase
```
vs
```
<?php
namespace PHP_CodeSniffer\Tests;
class TestSuite extends \PHPUnit_Framework_TestSuite
```

**Comment by darius-telycenas** [comment]
…or even like `https://github.com/squizlabs/PHP_CodeSniffer/blob/master/tests/AllTests.php`:
```
    /**
     * Add all PHP_CodeSniffer test suites into a single test suite.
     *
     * @return \PHPUnit_Framework_TestSuite
     */
    public static function suite()
    {
        $GLOBALS['PHP_CODESNIFFER_STANDARD_DIRS'] = array();
        $GLOBALS['PHP_CODESNIFFER_TEST_DIRS']     = array();
        // Use a special PHP_CodeSniffer test suite so that we can
        // unset our autoload function after the run.
        $suite = new TestSuite('PHP CodeSniffer');
        $suite->addTest(Core\AllTests::suite());
        $suite->addTest(Standards\AllSniffs::suite());
        return $suite;
}
```

**Pull request merged**